tutorials/getting_started/using_as_a_library_pt1.md

```diff
@@ -233,12 +225,7 @@ you need to do two things.  First, you need to _register_ your objects with our
 our code can find your class when it tries to instantiate a `DatasetReader`.  That is what the
 first line in the code above is doing; we register `SemanticScholarDatasetReader` as a
 `DatasetReader` with the name `s2_papers`, which will let us use that name in our JSON
-configuration file.  Then, when AllenNLP is trying to construct a `DatasetReader`, it will take the
-parameters you specify and pass them to the `from_params` method.  This method takes a `Params`
-object, which is just a JSON dictionary with some added functionality, and constructs the
-`SemanticScholarDatasetReader`.  All of the `DatasetReader`'s dependencies that we want to be able
-to configure from the JSON file need to be constructed here.  In this case, we just create a
-`Tokenizer` and a `TokenIndexer` dictionary, using methods that are built in to the library.
+configuration file.
```

Comment:
`from_params` is mentioned again below when discussing the config file.  Not sure what the best way to update that section is, though...